authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,63 @@
-# import jwt
-#
+import datetime
+import logging
+
+import jwt
+import os
+
 # from django.conf import settings
 #
 # from rest_framework import authentication, exceptions
 #
 # from .models import User
+from django.conf import settings
 
 """Configure JWT Here"""
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+
+# Get an instance of a logger
+logger = logging.getLogger(__name__)
+
+
+class JWTAuthentication(JSONWebTokenAuthentication):
+    """Inherit the JSON web authentication class from rest_framework_jwt"""
+
+    # @staticmethod
+    # def encode_token(obj):
+    #     jwt_payload_handler = api_settings.JWT_PAYLOAD_HANDLER
+    #     jwt_encode_handler = api_settings.JWT_ENCODE_HANDLER
+    #
+    #     payload = jwt_payload_handler(obj)
+    #     token = jwt_encode_handler(payload)
+    #     return token
+    @staticmethod
+    def encode_token(user, is_refresh_token=False):
```

Comment:
I feel like the function name is not descriptive and a bit misleading. Kindly consider changing that to `generate_token`

Reply:
That makes sense. Am working on it.